proxy/coco/src/peer/run_state.rs

```diff
@@ -112,10 +128,12 @@ pub enum Status {
     /// Phase where the local peer tries get up-to-date.
     #[serde(rename_all = "camelCase")]
     Syncing {
-        /// Number of completed syncs.
-        synced: usize,
-        /// Number of synchronisation underway.
-        syncs: usize,
+        /// The set of failed syncs.
```

Comment:
We now keep track of the `PeerId`s we've interacted with for the syncing process on startup